From 61de25a271b897fbc42e5683b7bcbb29de615123 Mon Sep 17 00:00:00 2001
From: Rahul Sandhu <nvraxn@gmail.com>
Date: Thu, 26 Jun 2025 11:15:00 +0100
Subject: [PATCH 1/2] autotools: concatenate $CXXFLAGS in a posix compliant
 manner

The += operator is not posix compliant and on systems where /bin/sh
points to a posix compliant shell, CXXFLAGS+= will be treated as a
command to execute:

./configure: 19608: CXXFLAGS+= -std=c++11: not found
./configure: 19609: CXXFLAGS+= -pedantic: not found
./configure: 19610: CXXFLAGS+= -Wno-unknown-pragmas: not found
./configure: 19611: CXXFLAGS+= -Wall: not found
./configure: 19612: CXXFLAGS+= -Wextra: not found

Signed-off-by: Rahul Sandhu <nvraxn@gmail.com>
---
 configure.ac | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index a867ee6..aa7e588 100644
--- a/configure.ac
+++ b/configure.ac
@@ -140,11 +140,11 @@ AC_ARG_ENABLE(
 )
 
 CXXFLAGS=" $EXTERNAL_CXXFLAGS"
-CXXFLAGS+=" -std=c++11"
-CXXFLAGS+=" -pedantic"
-CXXFLAGS+=" -Wno-unknown-pragmas"
-CXXFLAGS+=" -Wall"
-CXXFLAGS+=" -Wextra"
+CXXFLAGS="$CXXFLAGS -std=c++11"
+CXXFLAGS="$CXXFLAGS -pedantic"
+CXXFLAGS="$CXXFLAGS -Wno-unknown-pragmas"
+CXXFLAGS="$CXXFLAGS -Wall"
+CXXFLAGS="$CXXFLAGS -Wextra"
 
 if test "x$debug" = xyes; then
     CXXFLAGS="$CXXFLAGS $CXXFLAGS_DEBUG"
-- 
2.50.0

